### DISCUSSION TITLE
0xHACK Hackathon Winner!

### DISCUSSION THREAD

**Discussion by xandronus** [xandronus/linkfit#2] [General]
Pleased to announce that this project won both the bounties it submitted for in the 0xHACK hackathon. The announcement can be seen here (near the end) https://youtu.be/50FG4zhxuzc 

Happy Hacking!

**Comment by tippi-fifestarr** [comment]
Hey!  You did amazing work!  I was the Proof-of-Workout guy, can we find a chance to chat?

**Reply by xandronus** [reply]
Thanks! I haven't had a chance to check out your project yet but will try to soon. If you want to get a hold of me you can DM me on twitter @xandronus  or email me xandronus@gmail.com
